pkg/dashboard/handlers/helmHandlers.go

```diff
@@ -176,6 +176,22 @@ func (h *HelmHandler) Install(c *gin.Context) {
 	c.String(http.StatusAccepted, out)
 }
 
+func (h *HelmHandler) Tests(c *gin.Context) {
+	qp, err := utils.GetQueryProps(c, false)
+	if err != nil {
+		_ = c.AbortWithError(http.StatusBadRequest, err)
+		return
+	}
+
+	out, err := h.Data.RunTests(qp.Namespace, qp.Name)
+	if err != nil {
+		_ = c.AbortWithError(http.StatusInternalServerError, err)
+		return
+	}
+
+	c.String(http.StatusAccepted, out)
```

Comment:
The tests can actually have output and logs. Shouldn't we block until those are returned, and show the result in UI?

Reply:
I mean `StatusAccepted`  has meaning "the request is accepted for processing, but it is not complete. If it is complete, the status code should be 200 OK

Reply:
Handled in new commit